client/src/app/employee/employee-list/employee-list.component.ts

```diff
@@ -13,21 +10,20 @@ import { EmployeeListService } from './state/employee-list.service';
 export class EmployeeListComponent implements OnInit, OnDestroy {
   readonly faSkull = faSkull;
   readonly faSmileBeam = faSmileBeam;
-  readonly employees: Observable<Employee[]>;
 
-  constructor(
-    private service: EmployeeListService,
-    private query: EmployeeListQuery,
-  ) {
-    this.employees = this.query.selectAll();
-    this.service.getEmployees();
-  }
+  @Input()
+  employees: EmployeeListItem[];
+
+  @Output()
+  selected = new EventEmitter<EmployeeListItem>();
+
+  constructor() { }
 
-  ngOnInit() {}
+  ngOnInit() { }
```

Comment:
In most presentational components you will find you don't need `ngOnInit()` or `ngOnDestroy()` because there's no special Angular things you are doing with them - they just bind `@Input()` state to the template.

Reply:
Fixed this throughout multiple components.